src/basic/efivars.c

```diff
@@ -353,9 +362,40 @@ int systemd_efi_options_variable(char **line) {
 
                 *line = m;
                 return 0;
+        } else {
+                r = read_one_line_file(EFIVAR_CACHE_PATH(EFI_VARIABLE_SystemdOptions), line);
+                if (r == -ENOENT)
+                        return -ENODATA;
+                return r;
+        }
+}
+
+static inline int compare_stat_mtime(const struct stat *a, const struct stat *b) {
+        return CMP(a->st_mtim.tv_sec, b->st_mtim.tv_sec) ?:
+               CMP(a->st_mtim.tv_nsec, b->st_mtim.tv_nsec);
```

Comment:
i'd prefer:

```c
return CMP(timespec_load(&a->st_mtim), timespec_load(&b->st_mtim));
```

It's shorter and easier to read.

Reply:
Yep.